.github/workflows/lint-rust.yml

```diff
@@ -0,0 +1,67 @@
+name: Lint Rust
+
+on:
+  pull_request:
+    paths:
+      - polars/**
+      - polars-cli/**
+      - examples/**
+      - Cargo.toml
+      - .github/workflows/lint-rust.yml
+  push:
+    branches:
+      - main
+    paths:
+      - polars/**
+      - polars-cli/**
+      - examples/**
+      - Cargo.toml
+      - .github/workflows/lint-rust.yml
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Rust
+        run: rustup component add rustfmt clippy miri
+
+      - name: Cache Rust
+        uses: Swatinem/rust-cache@v2
+        with:
+          save-if: ${{ github.ref_name == 'main' }}
+
+      - name: Run cargo fmt
+        if: ${{ github.ref_name != 'main' }}
+        run: cargo fmt --all -- --check
+
+      - name: Run cargo clippy on all features
+        run: cargo clippy --all-features -- -D warnings
+
+      - name: Run cargo clippy
+        if: ${{ github.ref_name != 'main' }}
+        run: cargo clippy -- -D warnings
```

Comment:
shouldn't we keep the `-C debuginfo=0` flag?

Reply:
I read [here](https://doc.rust-lang.org/rustc/codegen-options/index.html#debuginfo) that `debuginfo=0` is the default, so I'm not sure why these compiler flags are included...

Reply:
Nice! Thanks for the info. I'll update accordingly.